src/glcontrol/cli.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env -S poetry run python
+import os
+import time
+
+import typer
+from framegrab.cli.clitools import preview_image
+
+from glcontrol.cfgtools.specs import GLControlManifest
+from glcontrol.runner import SpecRunner
+
+app = typer.Typer()
```

Comment:
I don't know why this isn't the default in typer, but the following change will make "-h" and "--help" both work for the help message.

```suggestion
app = typer.Typer(context_settings={"help_option_names": ["-h", "--help"]})
```